symphony/content/content.blueprintsdatasources.php

```diff
@@ -331,7 +331,7 @@ public function __form()
         $fieldset = new XMLElement('fieldset');
         $this->setContext($fieldset, array('sections', 'system'));
         $fieldset->appendChild(new XMLElement('legend', __('Error Conditions')));
-        $p = new XMLElement('p', __('Enabling these conditions will send a <code>404 page not found</code> error.'));
+        $p = new XMLElement('p', __('If enabled conditions are met, a <code>"404 page not found"</code> error will be sent.'));
         $p->setAttribute('class', 'help');
```

Comment:
I agree that responses are sent and received. This is done by the server resp. "the software", normally. But currently we say that "meeting a condition" will send a response, which sounds really strange to my ears. How can "meeeting a condition" actively send something? It can only cause the software to do it. (That's why I used the verb "yield".)

I would like to hear what native speakers think about it.